src/sentry/integrations/pipeline.py

```diff
@@ -255,16 +255,11 @@ def _finish_pipeline(self, data):
                 extra={
                     "organization_id": self.organization.id if self.organization else None,
                     "provider_key": self.provider.key,
+                    "integration_id": self.integration.id,
```

Comment:
Just curious, why do you need to add `integration_id` here to change the response?

Reply:
we don't this addition just makes my life easier to debug when cust are going why can't I install Jira.